explorer/scan/local_scanner.go

```diff
@@ -272,6 +273,7 @@ func (s *LocalScanner) distributeJob(job *Job, ctx context.Context, upstream *up
 			Asset:    candidate.asset,
 			Upstream: upstream,
 		})
+		candidate.asset = runtime.Provider.Connection.Asset // to ensure we get all the information the connect call gave us
```

Comment:
do we still need this with the changes on line 215?

Reply:
we did when i tested - because one is for that main clires asset and the other is for the candidate assets

Reply:
ooh, i see